documentation/pvDatabaseJava.html

```diff
@@ -36,29 +36,8 @@
 
 <div class="head">
 <h1>pvDatabaseJava</h1>
-<!-- Maturity: Working Draft or Request for Comments, or Recommendation, and date. -->
+<h2 class="nocount">Release 4.2 - 2016.03.21</h2>
```

Comment:
@dhickin proposed to use the ...-DEV notation here, to clarify that this is documentation in progress,